src/tools/roc_recv/cmdline.ggo

```diff
@@ -92,6 +92,9 @@ FILE_FORMAT is the output file format name, e.g.:
 TIME is an integer number with a suffix, e.g.:
   123ns; 123us; 123ms; 123s; 123m; 123h;
 
+SIZE is an integer number with an optional suffix, e.g.:
+  123M; 123K; 123;
```

Comment:
Let's document "G" too. It doesn't make sense for frame sizes, but may be useful for other limits (e.g. max total memory).